### ISSUE TITLE
Release/Git Tags

### ISSUE THREAD

**Issue by bmgandre** [matt-42/iod#13]
It's been two years since release 1.0 and I see here <https://github.com/matt-42/silicon/commit/2894f0691c6d19e0bf0b7e25be55149bdb281f72> that there is a new version for iod. 
Could you provide/update repo tags?

**Comment by matt-42** [comment]
Yes. I'll release a new version of iod soon, probably in January.
